src/utils/fs.rs

```diff
@@ -59,7 +59,7 @@ pub fn cd_into_same_dir_as(filename: &Path) -> crate::Result<PathBuf> {
 }
 
 /// Try to detect the file extension by looking for known magic strings
-/// Source: https://en.wikipedia.org/wiki/List_of_file_signatures
+/// Source: <https://en.wikipedia.org/wiki/List_of_file_signatures>
```

Comment:
Did you forgot this one?

Reply:
No, only the doc comments in _src/opts.rs_ are affected since they are the only ones that get used by clippy to generate help output. This and the URL in _src/utils/formatting.rs_ are purely internal documentation, therefore we can make use of the rustdoc automatic link markup here :)

Reply:
Oh yeah xD my bad.